packages/calcite-components/src/components/input-date-picker/input-date-picker.scss

```diff
@@ -1,55 +1,82 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+
```

Comment:
For consistency, can you make sure all lines in the JSDoc block start with a `*`?